sql/core/src/test/resources/sql-tests/results/subquery/in-subquery/in-group-by.sql.out

```diff
@@ -172,16 +172,16 @@ WHERE  t1b IN (SELECT Max(t2b)
                FROM   t2
                GROUP  BY t2a)
 -- !query 9 schema
-struct<t1a:string,t1b:smallint,t1c:int,t1d:bigint,t1e:float,t1f:double,t1g:decimal(2,-2),t1h:timestamp,t1i:date>
+struct<t1a:string,t1b:smallint,t1c:int,t1d:bigint,t1e:float,t1f:double,t1g:double,t1h:timestamp,t1i:date>
```

Comment:
let's update the CREATE TABLE statements in this test files to create decimal column correctly. It tries to test both double and decimal type and we should keep the test coverage.